docs/source/guide/admin_permissions.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Customize organization permissions
+short: Permissions
+tier: enterprise
+type: guide
+hide_menu: false
+order: 0
+order_enterprise: 362
+meta_title: Customize organization permissions
+section: "Manage Your Organization"
+parent_enterprise: "admin_settings"
+date: 2025-02-18 12:03:59
+---
+
+!!! note
+    This page is only visible to users in the Owner role. 
+
+You can use this page to customize certain permissions for roles. For information about Label Studio Enterprise roles and default permissions, see [User roles and permissions](admin_roles).
+
+Note the following:
+
+* Permissions for the Owner role are not configurable. 
+* You can use the options on this page to restrict permissions, but you cannot extend permissions. 
+
+    For example, if an action is typically available to Managers, Admins, and Owners, you can restrict it to Admins and Owners, but you cannot *extend* it to Annotators or Reviewers. 
+
+* Restrictions will also apply to the API. For example, if you restrict Managers' ability to configure cloud storage, they will not be able to complete those actions through the UI or the API. 
+
+
+Permissions that can be restricted include the following:
+
+| Permission | Default Roles |
```

Comment:
Is there any way we can build this list dynamically, based on a list coming from the code itself? I know we will forget to update this when adding new configurable permissions. I know we have this problem in other places, wondering if you have any ideas @caitlinwheeless

Reply:
(I agree *we* will forget, but I doubt Caitlin will :)